services/kvstore/src/main/java/tech/pegasys/pantheon/services/kvstore/RocksDbConfiguration.java

```diff
@@ -108,6 +115,15 @@ public String getLabel() {
         description = "Number of RocksDB background threads (default: ${DEFAULT-VALUE})")
     int backgroundThreadCount;
 
+    @CommandLine.Option(
+        names = {"--Xrocksdb-columns-enabled"},
+        hidden = true,
+        defaultValue = "true",
+        paramLabel = "<BOOLEAN>",
+        description =
+            "Whether to separate chain and world state into separate RocksDB columns (default: ${DEFAULT-VALUE})")
+    boolean useColumns = true;
```

Comment:
Since we haven't done the versioning as part of this PR (good choice), we need to set the default to `false`. :)

Once we do have database versioning, we should change this to allow specifying a version rather than just toggling columns on/off.  That will be quite a useful toggle for testing in the future I suspect.